demonstrations/tutorial_classical_kernels.py

```diff
@@ -0,0 +1,738 @@
+r"""
+
+.. _classical_kernels:
+
+How to approximate a classical kernel with a quantum computer
+=======================================================================
+
+.. meta::
+    :property="og:description": Finding a QK to approximate the Gaussian
+    kernel.
+    :property="og:image": https://pennylane.ai/qml/_images/toy_qek.png
+
+.. related::
+
+   tutorial_kernels_module Training and evaluating quantum kernels
+   tutorial_kernel_based_training Kernel-based training of quantum models with
+   scikit-learn
+   tutorial_expressivity_fourier_series Quantum models as Fourier series
+
+
+*Author: Elies Gil-Fuster (Xanadu resident). Posted: DD MMMM 2021.*
+
+Forget about advantages, supremacies, or speed-ups.
+Let us understand better what we can and cannot do with a quantum computer.
+More specifically, in this demo we want to look into quantum kernels and ask
+whether we can replicate classical kernel functions with a quantum computer.
+Lots of researchers have lengthily stared at the opposite question, namely that
+of classical simulation of quantum algorithms.
+Yet, by studying what classes of functions we can realize with quantum kernels,
+we can gain some insight into their inner workings.
+
+Usually, in quantum machine learning (QML) we use parametrized quantum circuits
+(PQCs) to find good functions, whatever *good* means here.
+Consequently, since kernels are just one specific kind of well-defined
+functions, the task of finding a quantum kernel (QK) that approximates a given
+classical one could be posed as a supervised learning regression problem.
+The way to attack this task would be to define a loss function directly
+quantifying the distance between both functions (the classical kernel function,
+and the PQC-based hypothesis).
+This sort of approach is more or less oblivious to the Fourier representation
+of the kernels, though.
+
+We want to study the link between classical and quantum kernels through the
+lens of the Fourier representation of a kernel, which is a common tool in
+classical ML.
```

Comment:
```suggestion
In order to build intuition, we will instead study the link between classical and quantum kernels through the
lens of the Fourier representation of a kernel, which is a common tool in
classical ML.
```

Does that link it well?

Reply:
yes I think so!